test/test.mk

```diff
@@ -79,8 +78,9 @@ test/automated-run:
 	make test/automated/packaging
 
 .PHONY: test/runner/provision
+test/runner/provision: VERSION ?= "HEAD"
```

Comment:
allow overriding branch to fetch in the runner

Reply:
I would rename this to GIT_REF or something more uniq than just version

Reply:
Thanks, I though a couple of names and I didn't like any 😅 
Changed in d0efbc8d